### ISSUE TITLE
Bug: Switching from seats to door gunner opens help menu

### ISSUE THREAD

**Issue by BroBeansCPG** [Project-Hatchet/Interaction-Framework#37]
As described in Title 

Hop in back cargo seat, switch to door gunner. Help menu is opened ... 

Reproduced in mh60m

**Comment by Riverman02** [comment]
Fixed in PR #457 in the 60?

**Comment by ampersand38** [comment]
It's an H-60 issue anyway, right?

**Comment by BroBeansCPG** [comment]
> It's an H-60 issue anyway, right?

Not exactly, not sure if I fixed it when I fixed the crashing on seat change. (Seat switch eventhandler was broken) moved to the cba event which should capture ace swapping seats. 


Basically the issue is the framework not running modules on seat change in some circumstances, so it wasn’t hiding the help ui.
